packages/router-plugin/tests/delete-nodes/snapshots/react/1-delete-nodes-undefined/development/createRootRoute-ssr-function.tsx

```diff
@@ -0,0 +1,20 @@
+import { createRootRoute } from '@tanstack/react-router';
+import crypto from 'node:crypto';
+export const Route = createRootRoute({
+  ssr: () => {
+    if (crypto.randomInt(0, 2) === 0) {
+      return 'data-only';
+    }
+    return false;
+  },
+  component: () => {
+    return <div className="p-2">hello world</div>;
+  }
+});
+if (import.meta.hot) {
```

Comment:
Is it necessary to have both `development` and `production` snapshots related testing for the "delete-nodes" functionality?

Currently, the difference between 'development' and 'production', is the inclusion of the HMR setup using `import.meta.hot.accept`.